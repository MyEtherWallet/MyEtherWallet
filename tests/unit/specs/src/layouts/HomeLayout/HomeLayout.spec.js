xdescribe('HomeLayout.vue', () => {
  test('resnders correctly', () => {});
});
