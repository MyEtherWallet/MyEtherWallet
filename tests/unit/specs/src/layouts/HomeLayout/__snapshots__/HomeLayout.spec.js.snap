// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeLayout.vue resnders correctly 1`] = `
<div
  class="home"
>
  
  HELLO

</div>
`;
